src/qt/mwebpegoutdialog.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2020-2021 The Litecoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <qt/mwebpegoutdialog.h>
+#include <qt/forms/ui_mwebpegoutdialog.h>
+
+#include <qt/bitcoinunits.h>
+#include <qt/guiutil.h>
+#include <qt/optionsmodel.h>
+#include <qt/platformstyle.h>
+#include <qt/walletmodel.h>
+
+#include <bech32.h>
+#include <key_io.h>
+#include <wallet/wallet.h>
+
+#include <QAbstractButton>
+#include <QDialogButtonBox>
+
+MWEBPegOutDialog::MWEBPegOutDialog(const PlatformStyle *_platformStyle, QWidget *parent) :
```

Comment:
Same comments as PegIn dialog. This looks really good, but we lose a lot of functionality. The logic we had in mind was:
The logic should work like this:
* When the user clicks on Peg-Out LTC, the button should stay depressed, and the "Pay To" field should be updated to say something like `Peg-Out: MQJP3gNpTF9JAhEJDW87WfbyKsM3fxaSfW` and marked read-only.
* If they decide they don't want to peg-out, they can just click the button again, and it will change to unpressed, and Pay To will be cleared and editable.

I'm sorry this wasn't very clear in the figma. If you want to hop on a call sometime to go over the UI, I'd be happy to make time.